training-maquetado/scss/_titles.scss

```diff
@@ -0,0 +1,18 @@
+@import "_styles";
+
+$title-decoration-height: 90px;
+
+$title-padding-left: 51px;
```

Comment:
we don't usually use variables for all the sizes. We only set the values to a variable if it will be used in another part of the project. 

For example, if you add a size to the navbar or the future then you will save that value in a variable as you may need it in the future. In that case you can add a file `_sizes.scss` and set all those items there.

In this case you can directly pass the value to the element
`padding-left: 51px`

Also 1px usually doesn't make any difference so we tend to use more rounded numbers like 50px in this case